package.json

```diff
@@ -138,7 +140,7 @@
     "@babel/preset-stage-0": "7.0.0",
     "autoprefixer": "9.1.5",
     "babel-7-plugin-root-import": "5.4.0",
-    "babel-eslint": "10.0.1",
```

Comment:
Why the downgrades?

Reply:
@tisto why is this?
`"babel-eslint": "9.x",`
it's some requirement? Why not the last one?

Reply:
@sneridagh my mistake.I fixed it.